src/app/trainees/state/trainees.actions.ts

```diff
@@ -26,3 +26,16 @@ export class SearchTrainees {
   static readonly type = "[Trainees] Search";
   constructor(public query: string) {}
 }
+
+export class PaginateTrainees {
+  static readonly type = "[Trainees] Page";
+  constructor(public pageIndex: number) {}
```

Comment:
just wondering: shouldn't there also be a, number of items per page? i.e. pageIndex and pageSize parameters? i.e. if we're defaulting to pageSize = 20

Reply:
the `pageSize` of 20 is passed in via an input on the `<mat-paginator>`

Reply:
says pageSize=1 on <mat-paginator>

Reply:
And this is not passed to Uzairs code to return 20 items? Thats what I was alluding to

Reply:
ah yea my bad, good spot. just updated the pr with 20